src/components/chip-set/chip-set.tsx

```diff
@@ -62,6 +62,15 @@ export class ChipSet {
     @Prop({ reflect: true })
     public label: string;
 
+    /**
+     * Optional helper text to display below the chipset.
+     * When type is not `input`, the helper text is always displayed
+     * if the device is touch screen; otherwise it is shown when chip-set
+     * is hovered.
```

Comment:
And when the type _is_ `input`?

Reply:
> otherwise it is shown when chip-set is hovered.

Maybe it wasn't so clear after all 😄 

It's actually a bit incomplete too. Because that'll be shown when chipset is hovered or focused with keyboard navigation. I'll add a fixup.

thanks for the feedback.

Reply:
Thanks for the feedback. It was unclear.. I'll add a fixup soon

Reply:
I assume that when the type is input, the helper text just works the way it does for a limel-input-field, right? If so, I think it's ok to just write that. This comment doesn't have to describe that behaviour. It just needs to not seem like a part of the comment is missing, because that will confuse people 😄

Reply:
I was too quick in responding. 😄 changed my answer later, but I think you already saw it. 

I see what you mean. I've pushed a fixup now. Do you think I should simplify the comment and instead make an example? Or is it OK as it is?